lora/redis/streams.go

```diff
@@ -68,8 +70,11 @@ func NewEventStore(svc lora.Service, client *redis.Client, consumer string, log
 	}
 }
 
-func (es eventStore) Subscribe(subject string) {
-	es.client.XGroupCreateMkStream(stream, group, "$").Err()
+func (es eventStore) Subscribe(subject string) error {
+	if err := es.client.XGroupCreateMkStream(stream, group, "$").Err(); err != nil && err.Error() != exists {
+		return err
+	}
```

Comment:
You shouldn't return here. If consumer group already exists, just continue.

Reply:
@dusanb94 what we do here? I thought that this is the fix

Reply:
You return only if there is an error other than `BUSYGROUP Consumer Group name already exists`. If there is no error or an error is exactly this one, you will proceed with your subscription.

Reply:
Right, this is correct @anovakovic01

Reply:
Didn't see that part. IMHO, this line is too long. Can you break it into multiple lines? Also, is there predefined error for this in redis lib?

Reply:
I could not find any meaningful error. Line length fixed.